java-surefire/src/main/java/org/sonar/plugins/surefire/api/SurefireUtils.java

```diff
@@ -28,46 +28,95 @@
 import javax.annotation.CheckForNull;
 
 import java.io.File;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
 
 /**
  * @since 2.4
  */
 public final class SurefireUtils {
 
   private static final Logger LOGGER = Loggers.get(SurefireUtils.class);
+  /**
+   * @deprecated since 4.11
+   */
+  @Deprecated
   public static final String SUREFIRE_REPORTS_PATH_PROPERTY = "sonar.junit.reportsPath";
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)